### DISCUSSION TITLE
Different icons for spaces that are full, empty or focused

### DISCUSSION THREAD

**Discussion by itaysharir** [FelixKratz/SketchyBar#302] [Q&A] [answered]
Can a way to do that be added? Is it already possible and i missed it? Thanks in advance.

**Comment by FelixKratz** [comment, marked as answer]
This can be achieved already via a script for the space, where a query to yabai can be used to decide the icon of that space.
See my dotfiles, there I additionally choose to display all the active application icons in the space `label`:
https://github.com/FelixKratz/dotfiles/tree/master/.config/sketchybar

**Reply by itaysharir** [reply]
Thanks! your dotfiles helped me quite a lot

**Comment by itaysharir** [comment]
Also, can i change other aspects of the icon in the selected/full space like font size?

**Reply by FelixKratz** [reply]
You can change every property at any time in a script or from the commandline directly, just like you do in the initial configuration. All the changes are applied immediately or can be animated to. You can use the event system to subscribe to system events, or you have a script that changes properties on a click, or you add custom events and trigger them elsewhere, just as I have done for e.g. the event that updates the active application icons for the spaces in my dotfiles.
